vach/src/global/header.rs

```diff
@@ -140,15 +140,20 @@ impl Header {
 	/// ### Errors
 	///  - `io` errors
 	pub fn from_handle<T: Read>(mut handle: T) -> InternalResult<Header> {
-		let mut buffer = [0x69; Header::BASE_SIZE];
+		#![allow(clippy::uninit_assumed_init)]
+		// We are never reading from `buffer`, so it's safe to use uninitialized memory. We initialize it instantly after
+
+		use std::mem::MaybeUninit;
+		let mut buffer: [u8; Header::BASE_SIZE] = unsafe { MaybeUninit::uninit().assume_init() };
```

Comment:
Is this *minor* speedup **really** worth using unsafe code? Imo it's not, and I'd rather avoid unsafe as much as possible.

Reply:
It;s completely safe, since we never read from the uninitialized buffer

Reply:
How much faster is it though?